<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Detector</class>
 <widget class="QWidget" name="Detector">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>542</width>
    <height>773</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>10</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Название датчика:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1" colspan="2">
    <widget class="QLineEdit" name="nameEdit"/>
   </item>
   <item row="1" column="0">
    <widget class="QSplitter" name="splitter_2">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Выбор датчика:</string>
      </property>
     </widget>
     <widget class="QComboBox" name="cmbDetector"/>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="QTableWidget" name="tb1"/>
   </item>
   <item row="2" column="0" rowspan="8" colspan="2">
    <widget class="QSplitter" name="splitter">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>Выбор файла:</string>
      </property>
     </widget>
     <widget class="QListWidget" name="listFile"/>
    </widget>
   </item>
   <item row="2" column="2">
    <widget class="QRadioButton" name="rb1">
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Выбрать таблицу</string>
     </property>
    </widget>
   </item>
   <item row="3" column="2">
    <widget class="QTableWidget" name="tb2"/>
   </item>
   <item row="4" column="2">
    <widget class="QRadioButton" name="rb2">
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Выбрать таблицу</string>
     </property>
    </widget>
   </item>
   <item row="5" column="2">
    <widget class="QTableWidget" name="tb3"/>
   </item>
   <item row="6" column="2">
    <widget class="QRadioButton" name="rb3">
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Выбрать таблицу</string>
     </property>
    </widget>
   </item>
   <item row="7" column="2">
    <widget class="QTableWidget" name="tb4"/>
   </item>
   <item row="8" column="2">
    <widget class="QRadioButton" name="rb4">
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Выбрать таблицу</string>
     </property>
    </widget>
   </item>
   <item row="9" column="2">
    <widget class="QTableWidget" name="tb5"/>
   </item>
   <item row="10" column="2">
    <widget class="QRadioButton" name="rb5">
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Выбрать таблицу</string>
     </property>
    </widget>
   </item>
   <item row="11" column="0" colspan="3">
    <widget class="QSplitter" name="splitter_3">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <widget class="QSplitter" name="splitter_4">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QPushButton" name="write_one">
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Записать</string>
       </property>
      </widget>
      <widget class="QPushButton" name="save">
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Сохранить</string>
       </property>
      </widget>
      <widget class="QPushButton" name="delete_one">
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Удалить</string>
       </property>
      </widget>
     </widget>
     <widget class="QPushButton" name="loadFromJson">
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Загрузить из файла</string>
      </property>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
